### DISCUSSION TITLE
3.1.0: Unable to install on ChromeOS

### DISCUSSION THREAD

**Discussion by eppineda** [vercel/hyper#5761] [Q&A]
Congratulations on the release of 3.1.0!

Via the ChromeOS desktop, it says it cannot retrieve the app info:

![Screenshot 2021-07-18 08 27 56](https://user-images.githubusercontent.com/6559491/126067041-63247b2a-f2c3-4e7a-a8c6-a105764411a2.png)

Via the command line, some errors about conflicting signing info:

![Screenshot 2021-07-18 08 16 49](https://user-images.githubusercontent.com/6559491/126066945-3ecd2c08-0395-4313-a4d9-174e2ea774e1.png)

I have a Google Pixel Slate, running Version 91.0.4472.114 (Official Build) (64-bit) of ChromeOS.

In advance, thank you for support and all you do with this wonderful utility.

**Comment by LabhanshAgrawal** [comment]
The linux builds are not signed
There's some issues with the deb downloaded from the site, refer to #5758 also.
